config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -10,7 +10,7 @@ presubmits:
     context: pull-security-kubernetes-e2e-aws-eks-1-11-correctness
     labels:
       preset-kubernetes-e2e-aws-eks-1-11: "true"
-      preset-kubernetes-e2e-aws-eks-common: "true"
+      preset-aws-credential: "607362164682"
```

Comment:
Btw - the label value is our testing account's ID. We can add new presets with different accounts.

Reply:
I suspect this will be error prone as nearly all of us (including me) don't have access to and have no idea what these are so we'll just be copying around mysterious number strings. I'd strongly suggest giving these a human readable name and commenting with the ID# on the preset instead.

Doing that would also mean you can swap accounts without mass updating all of the job config files, which is a big reason presets are useful.

Reply:
Something like:
`preset-aws-credential: cncf-prow`
`preset-aws-credential: aws-eks`

Which will also be useful because eventually @kubernetes/k8s-infra-wg will probably want to track usage on the CNCF credentials.

Reply:
Naming them with intent should also help job writers to figure out which they should be using from example jobs. The opaque strings will probably lead to someone creating a charts job on the wrong account or something.

Reply:
@BenTheElder Sounds reasonable.. Will change it to something more meaningful, thanks.

Reply:
Done, PTAL.